### DISCUSSION TITLE
Players need more things to do in Barotrauma (suggestions and discussion)

### DISCUSSION THREAD

**Discussion by Mantha065** [FakeFishGames/Barotrauma#9602] [Ideas and suggestions]
I REALLY enjoy this game, but I've played enough games to be able to sense when I start to get bored of one...and I can feel it coming on for Barotrauma. 
I've played 46 hours of the game. I've beaten the campaign with a multiplayer crew, I've gotten halfway through the campaign in a one-man sub, I've played every mission type, I've played as every role for many rounds each, I've explored beacons and wrecks, I've met every faction, and I've encountered most of the outpost events. It feels like the game doesn't offer anything beyond that.

Currently, there isn't much to do in any given playthrough: Combat with creatures and NPCs, completing a small variety of missions (Kill Stuff, Acquire Stuff, Transport Stuff, Fix Stuff), starting shenanigans in a multiplayer crew, maintaining and upgrading your sub, and encountering very short events at outposts. Granted, there's substantial replayability with each of these things, but if you get bored of these, there's nothing else the game contains for you. The gameplay is quite limited. However, I probably speak for many players when I say that this game has *lots* of potential, and I can easily see this game having much more depth to it...pun intended.

I understand this is still in Early Access and likely has a good few years left in the development phase, but I wanted to post this as a constructive critique and a gentle nudge to push for the full potential of this game. I've thought of some suggestions, anyone else who reads this can feel free to add to them:

-Overhaul of outpost NPCs; make each outpost feel like a unique location with unique people. More variety in dialogue, more events, and adding large-scale that take place across the whole station (say, your sub docks during a station-wide revolt and you must choose a side to fight with; or you arrive to find the station's reactor has melted down and you need to give people medical treatment and help repair the reactor; or a station has come under martial law, and you can help the military apprehend people or help the people convince the military to stand down)
-Add a chance to encounter a small NPC shuttle travelling through the level; you can ignore them, protect them from creatures, or attack, board, and loot the shuttle; each action gives or takes reputation from the outpost/faction they're associated with.
-Add a chance for a black market dealer to spawn inside an outpost, lurking away from the other NPCS; he can sell you rare or high-end items for a steep price.
-Add bandit stations you can dock at, if your reputation with all other factions is so poor that only fellow outlaws will accept you.
-Add an option to include a voting mechanic that lets players elect a new captain in multiplayer games; perhaps the crew mutinies, or the current captain is dead or has disconnected, etc.

There are so many interesting things you could add to the game, to make the world feel more lived-in, to give the player more things to do and more ways to play. I really hope this game fulfills its potential.

**Comment by mLuby** [comment]
> I've played 46 hours… It feels like the game doesn't offer anything beyond that.

That's just because you haven't found the Sub Editor yet. Join the Discord's #baro-sub-building channel and ask how many hours people there have played.

**Reply by Mantha065** [reply]
I don't really care about building subs. Even if I did...there's not much to do with them, is there? :P

**Comment by NilanthAnimosus** [comment]
so my thoughts here:

- I like the NPC Shuttles / friendly submarines idea. like pirate missions but surprise friendlies that you can attack or help ensure their not all dead by round end (Potential for reputation loss/gains here? Stolen items? etc.)

- Black market dealer, not a bad idea to have an extra kind of market...Perhaps tie relations to any purchases made from it (After all, its a black market) and be either cheaper goods or more expensive + rare?

- Perhaps it could be a server option to simply not limit job roles given the nature of campaigns and drop in/out, I don't think adding an entire mechanic to try to "Job change" or "force kill" by vote a player makes any sense or would be wanted by most campaigns and generally should be up to host discretion. in single player for instance we have no restrictions on how many captains or medics we can have/buy, but in campaigns we are limited on player assignment. and it doesn't cost any money...a tricky one in my eyes. voting a new captain though doesn't make sense as its like voting a new character change which you can already do in game (Just drop your stuff off and opt for a new character next round, its in the in-round menu somewhere to new character next round, such as when you want to re-roll to captain from a mechanic when your server has no captain).

- Bandit stations are interesting, but hopefully we get a framework instead that allows for different factions outpost-specific stuff. I would like to see clown cult outposts (they are a faction) as well as things like bandits, but given pirates are always hostile I don't think they should be based solely off the inverse of other factions (After all, both pirates and coalition can despise you). Perhaps introduce some alternative method to "Improve" pirate standings, or make them "Smuggler" faction outposts / faction (or just separatists) instead which act as a neutral site for pirates and none-pirates alike only being more tied to station-specific reputations. - if you manage to make every faction and station conceivable hate you to the point you cannot actually dock with stations or get missions anymore without gunshot wounds, I would consider this to be more a feature as it would have taken extreme attempts on the players side to achieve this, like murdering every outpost they encounter levels of impressive.

- The custom events like needing to repair a reactor or post-meltdown don't make much sense to me personally if you've ever triggered one of those to actually expload it typically takes the entire station along with it. and given the stations have large crews on hand (And presumably many, many, many more population off-screen as their...cities...outposts etc) I would expect them to have their own repair teams not requiring a single man from your submarine to use a screwdriver and wrench for 10 seconds to fix it. and the medical treatment...Maybe for some kind of plague or something but there isn't even a disease needing a medicine like this other than husk, by the time players notice this its usually too late with that one, and simply needing to hand over certain supplies for a "Plague" is kind of boring, and station staff should be able to heal people in stations on their own too usually...and the risks associated with something like helping a station put people down/into handcuffs is so high as hitting the wrong person is a very rapid way to have problems in campaign.

finally for more variety in dialogue, events etc...mods can add all of this in theory (Except the special scenarios above like starting off with a literal meltdown). no game generally stays in development forever so I hope they introduce more kinds of content and mechanics than just more raw events personally, Like AI crafting or picking AI talents, exposing to modding values like experience required for picking talents...or cool thrifty features like friendly submarine spawning in games.

content can always be modded in but new features does also mean more and better mods on top for them. if one is bored there are some good mods adding new spins on mission content, or overhauling the game to be harder/different.

**Comment by MaaiG** [comment]
IMHO, maybe we want a game world that’s alive enough to even want just watch to it?
And more reason for a full immersion in it?
- The player must have the feeling that the game is alive without his participation. Even if it is a lie ))
- Сosmos, radiation, gases, fluids, pressure - this environments is woven into the gameplay? Maximally?
- The physical game model gives you the ability to use it in the game to survive? Water displacing the oxygen? Pumping water out of a room automatically fills it with air? Can you play with these environments (liquids, gases, etc.)?
- How great is the diversity of opportunities for the player to use these environments to advance their stories and survive, sometimes at any cost? What are the ways to achieve the goals? What is the price and retribution for the short but immoral path? What is the reward for the long and honest one?
- There must be a lot of situations that will make the player think twice in order to survive?
- Even if your submarine is dead and you have managed to swim to the outpost, will you be able to complete the round?
- You finish the round and you have something to brag about? Can you tell the story of your survival and not lose listeners?
- If you were left without a submarine and no money, could you be a seaman or an officer on another NPC submarine?
- Can you make money for your own submarine or capture it and become a pirate?
- If you’re a vile criminal you can gas poison the whole team?
Could you be arrested and tried?
Can the crew take justice and throw you overboard as a traitor? Perhaps without a spacesuit!?
Are you going to survive?
- Does the game have a standard for rescue channels, military, engineering, security?
- You can create a channels scanner, but what are you going to look for?
- Your boat died, but you survived and rebuilt the radio station or were able to reach the lighthouse. Can you call for help and get it from the near outpost? Do you have a rescue or other submarine? Will you be rescued at least for money?
- Will you ever be able to descend to unprecedented depths and find your first dead submarine there?
- There are submarines in the game, but there is no dock for their maintenance and modification (that is, access to the editor directly from the game’s outpost with a shipyard).
- A pack of killer whales chase a whale and you witness these events?
- Sharks try to eat you, but something bigger comes at them and you get a chance to hide?
- Do you have the feeling that biome lives without you?
- Why do monsters always attack submarines? They don’t like her exhaust and instead of running, they attack him?
- The game has finances, but no bank to rob?
- There are prisoners, but there’s no way to be convicted and escape from a prison barge.
- Is there a Bermuda triangle and a strange ghost ship in the game?
- Electronic intelligence and counteraction?
- Remote access to devices?
- By repairing a silent beacon, can you tap into his CCTV system and try to see what’s inside?
- During a battle with the enemy in an abandoned outpost, you can get master control and control the doors and airlocks to try to drown the enemy or cut off his oxygen?
- The game is simply focused on joint shooting attacks against each other and nothing else?
- You’ve taken control of the station. Will the Corporation send the Special Forces to brutally punish you?
- What interesting stories are there in the game?
- What’s the main secret to the game?
- What have we learned?
- Playing a game is the same as reading an interesting book?
- What are you interested in?
- What are you thinking about it now?
- Did you rest your soul in this world or did your eyes just turn red?

If you are not deprived of some fantasy your expectations are always greater than what you can get?

*P.S. Of the well-known action games, the most interesting story is perhaps in Half-Life + Opposing Force + Blue Shift. The symbiosis of these three is just canonical! They are now visual obsolete, but the experiences they left behind will remain forever.
One can recall SOMA., Outlast and Outlast: Whistleblower and some etc.
All these games with a first-class story in a thoughtful world.
A good story is extremely important!*

**Reply by MateusJK15** [reply]
I'm sorry sir, but you're making too many questions and we need to eliminate you.

![Sem título](https://user-images.githubusercontent.com/86162808/180881536-8f8000bb-0475-44b8-a2b0-27f2a16af405.png)
